packages/mongodb-memory-server-core/src/util/__tests__/MongoBinary.test.ts

```diff
@@ -138,6 +138,32 @@ build environment:
         expect(output).toBe(sysBinaryPath);
       });
 
+      it('should return and check an SystemBinary and match even if binary has patch version', async () => {
+        // Output taken from mongodb x64 for "ubuntu" version "4.0.25"
```

Comment:
please update this comment to reflect the proper version

Reply:
done :smile: